contributors.d

```diff
@@ -102,6 +102,7 @@ auto reduceAuthors(GitAuthors)(GitAuthors authors)
             .uniq!((a, b) => a.name == b.name);
 }
 
+version(Contributors_Lib) {} else
```

Comment:
I didn't find a better way which allows to let both scripts be runnable without any flags passed and avoids creating additional files.